### PR TITLE
feat(textfield): add onClear callback to props

### PR DESCRIPTION
**Pull request by al3xbzh**
# General summary

Added a callback to be able to trigger some function when a user clicks on the clear button.

# Screenshots

**There are no visual updates in this PR**
clearable-textfield

### CODE REVIEW

File: packages/lumx-react/src/components/text-field/TextField.stories.tsx [hunk 1]
```diff
@@ -53,6 +53,7 @@ export const Clearable = {
     args: {
         value: 'Some value',
         clearButtonProps: { label: 'Clear' },
+        onClear: () => console.log('onClear called'),
```
Comment (gcornut) [inline packages/lumx-react/src/components/text-field/TextField.stories.tsx:56]: Can you use SB actions in argTypes instead ? It's a bit cleaner that way 
```
argTypes: { onClear: { action: true } },
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
```
Comment (gcornut) [inline CHANGELOG.md:10]: Can you rebase and merge changes with master ?

**Review by gcornut — CHANGES_REQUESTED** [review]
Thanks for the PR ! I only have 2 small comment :)

File: packages/lumx-react/src/components/text-field/TextField.stories.tsx (hunk 1 above)
Comment (al3xbzh) [reply packages/lumx-react/src/components/text-field/TextField.stories.tsx]: done ☑️

File: CHANGELOG.md (hunk 2 above)
Comment (al3xbzh) [reply CHANGELOG.md:10]: done ☑️

**Review by gcornut — APPROVED** [review]

**Comment by gcornut** [comment]
Testing on SB (chromatic deploy)...

**Pull request merged**